hover/effects/_keyframes.sass

```diff
@@ -0,0 +1,10 @@
+// Keyframes mixin to add browser prefixes to animations
+@mixin keyframes($animation-name)
+    @-webkit-keyframes #{$animation-name}
+        @content
+
+    @-moz-keyframes #{$animation-name}
```

Comment:
up to you really... but just checked - Firefox has full support since v16 so since a long time:)